homeassistant/components/zha/core/channels/base.py

```diff
@@ -195,42 +196,42 @@ async def configure_reporting(self) -> None:
         if self.cluster.cluster_id >= 0xFC00 and self._ch_pool.manufacturer_code:
             kwargs["manufacturer"] = self._ch_pool.manufacturer_code
 
-        for report in self._report_config:
-            attr = report["attr"]
+        for attr_report in self.REPORT_CONFIG:
+            attr, config = attr_report["attr"], attr_report["config"]
             attr_name = self.cluster.attributes.get(attr, [attr])[0]
-            min_report_int, max_report_int, reportable_change = report["config"]
             event_data[attr_name] = {
-                "min": min_report_int,
-                "max": max_report_int,
+                "min": config[0],
+                "max": config[1],
                 "id": attr,
                 "name": attr_name,
-                "change": reportable_change,
+                "change": config[2],
+                "success": False,
             }
 
+        to_configure = [*self.REPORT_CONFIG]
+        chunk, rest = (
+            to_configure[:REPORT_CONFIG_ATTR_PER_REQ],
+            to_configure[REPORT_CONFIG_ATTR_PER_REQ:],
+        )
+        while chunk:
+            reports = {rec["attr"]: rec["config"] for rec in chunk}
             try:
-                res = await self.cluster.configure_reporting(
-                    attr, min_report_int, max_report_int, reportable_change, **kwargs
-                )
-                self.debug(
-                    "reporting '%s' attr on '%s' cluster: %d/%d/%d: Result: '%s'",
-                    attr_name,
-                    self.cluster.ep_attribute,
-                    min_report_int,
-                    max_report_int,
-                    reportable_change,
-                    res,
-                )
-                event_data[attr_name]["success"] = (
-                    res[0][0].status == 0 or res[0][0].status == 134
-                )
+                res = await self.cluster.configure_reporting_multiple(reports, **kwargs)
+                self._configure_reporting_status(reports, res[0])
+                # if we get a response, then it's a success
+                for attr_stat in event_data.values():
+                    attr_stat["success"] = True
```

Comment:
Why was this behaviour changed here? Was reported an issue on here: https://github.com/home-assistant/core/issues/104810

Reply:
Def looks like a bug… success in this context should only be success (0) and unsupported (134)

Reply:
The reason we treat unsupported as success is we don’t want to flag configuration/reconfiguration as failed for unsupported attributes… the command itself was successfully executed as far as those processes go

Reply:
We cant mark them as none or something?